client/my-sites/site-settings/form-general.jsx

```diff
@@ -609,10 +618,38 @@ export class SiteSettingsFormGeneral extends Component {
 
 		const LaunchCard = showPreviewLink ? CompactCard : Card;
 
+		if ( isSiteOnECommerceTrial ) {
+			const recordTracksEventForClick = () =>
+				recordTracksEvent( 'calypso_ecommerce_trial_launch_banner_click' );
+			const eCommerceTrialUpsellText = translate(
+				'Before you can share your store with the world, you need to {{a}}pick a plan{{/a}}.',
+				{
+					components: {
+						a: (
+							<a
+								href={ `/plans/${ siteSlug }?plan=ecommerce-bundle-monthly` }
```

Comment:
We should probably use the plan constant (although I recognize this will probably change!) rather than a hard-coded value.